plat/layerscape/common/ls_gicv2.c

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
The plat_arm_* interfaces are internal porting interfaces for ARM reference platforms in plat/arm/ folder. This file is particularly used to abstract the common functionality between GICv2 and GICv3 platforms. If ls platform is only ever interested in GICv2, then this plat_arm_gic* abstraction may not be needed.

Also, I have my doubts regarding the GIC driver initialization.  The driver asserts that there is alteast 1 secure interrupt to configure. But the gicv2_driver_data defined here doesnt seem to be declaring any secure interrupt (see here: https://github.com/ARM-software/arm-trusted-firmware/blob/c37be00b160da5e8aa1a2950090fc52002d88d95/drivers/arm/gic/v2/gicv2_main.c#L172)